core/src/main/java/io/aiven/klaw/controller/AclController.java

```diff
@@ -66,6 +72,7 @@ public ResponseEntity<List<AclRequestsResponseModel>> getAclRequests(
       @RequestParam(value = "env", required = false) String env,
       @RequestParam(value = "search", required = false) String search,
       @RequestParam(value = "aclType", required = false) AclType aclType,
+      @RequestParam(value = "order", required = false, defaultValue = "NEWEST_FIRST") Order order,
```

Comment:
```suggestion
      @RequestParam(value = "order", required = false, defaultValue = "NEWEST_FIRST") Order orderByCreated,
```
i think order is on createddate

Reply:
Again here my main thought is we might add additional ordering options in the future so I didnt want to get into specifics here that might in the future make it a bit grey as to what the ordering is supposed to be.